scipy/stats/tests/test_qmc.py

```diff
@@ -650,18 +653,30 @@ def test_sample(self, *args):
         pytest.skip("Not applicable: the value of reference sample is"
                     " implementation dependent.")
 
+    def test_centering(self):
+        # centering stratifies samples in the middle of equal segments:
+        # * inter-sample distance is constant in 1D sub-projections
+        # * after ordering, columns are equal
+        rng = np.random.default_rng(230848485337593016117637089968181108379)
+        engine = qmc.LatinHypercube(d=3, scramble=False, seed=rng)
+        for _ in range(10):
+            sample = engine.random(100)
+            sorted_sample = np.sort(sample, axis=0)
+            mean_cols = np.mean(sorted_sample, axis=1)
```

Comment:
I'd suggest broadcasting the linspaced sample rather than taking the mean to clarify the intent and ensure that it really satisfies the property you're after.